go/ephemeral/device_ek.go

```diff
@@ -21,13 +21,16 @@ func newDeviceEphemeralSeed() (seed DeviceEKSeed, err error) {
 	return DeviceEKSeed(randomSeed), nil
 }
 
-func (s *DeviceEKSeed) DeriveDHKey() (key *libkb.NaclDHKeyPair, err error) {
-	derived, err := libkb.DeriveFromSecret(*s, libkb.DeriveReasonDeviceEKEncryption)
+func newDeviceEKSeedFromBytes(b []byte) (s DeviceEKSeed, err error) {
```

Comment:
Does this get called anywhere? Same q about the userEK version.

Reply:
devices can be removed, for users it's how we convert from the decrypted bytes to our type